File: Module.Frontend.TPM/Client/model/tpm/promoproductpriceincreasesview/PromoProductPriceIncreasesView.js
Ext.define('App.model.tpm.promoproductpriceincreasesview.PromoProductPriceIncreasesView', {
    extend: 'Ext.data.Model',
    idProperty: 'Id',
    breezeEntityType: 'PromoProductPriceIncreasesView',
    fields: [
        { name: 'Id', hidden: true },
        { name: 'ZREP', type: 'string', hidden: false, isDefault: true },
        { name: 'ProductEN', type: 'string', hidden: false, isDefault: true },
        { name: 'PlanProductBaselineLSV', type: 'float', hidden: false, isDefault: true, useNull: true },
        { name: 'PlanProductUpliftPercent', type: 'float', hidden: false, isDefault: true, useNull: true },
        { name: 'PlanProductIncrementalLSV', type: 'float', hidden: false, isDefault: true, useNull: true },
        { name: 'PlanProductLSV', type: 'float', hidden: false, isDefault: true, useNull: true },
        { name: 'PlanProductBaselineCaseQty', type: 'float', hidden: false, isDefault: true, useNull: true },
        { name: 'PlanProductIncrementalCaseQty', type: 'float', hidden: false, isDefault: true, useNull: true },
        { name: 'PlanProductCaseQty', type: 'float', hidden: false, isDefault: true, useNull: true },
        { name: 'SumInvoiceProduct', type: 'float', hidden: false, isDefault: false, useNull: true },
        {
            name: 'AverageMarker', type: 'bool', hidden: false, isDefault: true,
            convert: function (value) {
                return (value === true || value === 'Yes') ? l10n.ns('core', 'booleanValues').value('true') : l10n.ns('core', 'booleanValues').value('false');
            }
        },
        {
            name: 'IsCorrection', type: 'bool', hidden: false, isDefault: true,
            convert: function (value) {
                return (value === true || value === 'Yes') ? l10n.ns('core', 'booleanValues').value('true') : l10n.ns('core', 'booleanValues').value('false');
            }
        },
    ],

    proxy: {
        type: 'breeze',
        resourceName: 'PromoProductPriceIncreaseViews',
        reader: {
            type: 'json',
            totalProperty: 'inlineCount',
            root: 'results'
        },
        extraParams: {
            promoId: null
        }
    },
});

File: Module.Persist.TPM/Model/DTO/PromoProductPriceIncreasesView.cs
using Core.Data;
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Module.Persist.TPM.Model.DTO
{
    [Table("PromoProductPriceIncreasesView")]
    public class PromoProductPriceIncreasesView : IEntity<Guid>
    {
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public Guid Id { get; set; }
        [StringLength(255)]
        public string ZREP { get; set; }
        public string ProductEN { get; set; }
        public double? PlanProductBaselineLSV { get; set; }
        public double? PlanProductUpliftPercent { get; set; }
        public double? PlanProductIncrementalLSV { get; set; }
        public double? PlanProductLSV { get; set; }
        public double? PlanProductBaselineCaseQty { get; set; }
        public double? PlanProductIncrementalCaseQty { get; set; }
        public double? PlanProductCaseQty { get; set; }
        public bool AverageMarker { get; set; }
        public bool IsCorrection { get; set; }
    }
}
